### PR TITLE
feat: allow providers to expose more mimetypes than png+jpeg

### PR DESCRIPTION
**Pull request by tbuckley**

### CODE REVIEW

File: examples/files/runs/1726443642605.json
Comment (dstoc) [inline examples/files/runs/1726443642605.json:1]: add runs/ to .gitignore?

**Review by dstoc — APPROVED** [review]
I think We might want a way to override the supported set of mime types from the provider config.

File: examples/files/runs/1726443642605.json
Comment (tbuckley) [reply examples/files/runs/1726443642605.json:1]: Wanted to keep to both have an example output for users and check backwards compatibility via playwright tests.

**Comment by tbuckley** [comment]
> I think We might want a way to override the supported set of mime types from the provider config, because it can vary per model.

Added */* support for ollama. Agree we probably still want a way to force models to try using other file types if they are added

**Pull request merged**